tsconfig.base.json

```diff
@@ -1,6 +1,8 @@
 {
-	"extends": "@sapphire/ts-config",
+	"extends": ["@sapphire/ts-config", "@sapphire/ts-config/extra-strict", "@sapphire/ts-config/decorators", "@sapphire/ts-config/verbatim"],
 	"compilerOptions": {
+		"module": "NodeNext",
```

Comment:
We should be able to remove this. NodeNext is identical to Node16 which is used by sapphire by default.